<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnreportedStudies</class>
 <widget class="QMainWindow" name="UnreportedStudies">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Higiia - Study Pool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../icons.qrc">
    <normaloff>:/icons/favicon.ico</normaloff>:/icons/favicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font-size: 9pt;
}

 QLineEdit:!enabled {
     color: black;
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: lightgray;
     selection-background-color: darkgray;
 }


 QLineEdit {
     color: black;
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: white;
     selection-background-color: darkgray;
 }

 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
	color: black;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: white; /* make the default button prominent */
 }

QFrame{
	border: none;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" rowspan="2">
     <widget class="QFrame" name="frame_5">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="QLabel" name="lblSearchPatient">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../../icons.qrc">:/icons/icoSearch.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Search for Patient Name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="txtSearchPatient"/>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" rowspan="4">
           <widget class="QTableWidget" name="tblInfo">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btnBack">
            <property name="minimumSize">
             <size>
              <width>84</width>
              <height>65</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>65</width>
              <height>65</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../icons.qrc">
              <normaloff>:/icons/icoUpArrow.png</normaloff>:/icons/icoUpArrow.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btnNext">
            <property name="minimumSize">
             <size>
              <width>84</width>
              <height>65</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>65</width>
              <height>65</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../icons.qrc">
              <normaloff>:/icons/icoDownArrow.png</normaloff>:/icons/icoDownArrow.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="styleSheet">
          <string notr="true">QFrame{
	border: 0px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnClose">
            <property name="minimumSize">
             <size>
              <width>84</width>
              <height>93</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>93</width>
              <height>93</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../icons.qrc">
              <normaloff>:/icons/icoCloseReverseLine.png</normaloff>:/icons/icoCloseReverseLine.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnPACS">
            <property name="minimumSize">
             <size>
              <width>84</width>
              <height>93</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>84</width>
              <height>93</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>19</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget {
	font-size: 19pt;
}</string>
            </property>
            <property name="text">
             <string>PACS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnViewStudy">
            <property name="minimumSize">
             <size>
              <width>84</width>
              <height>93</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>93</width>
              <height>93</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../icons.qrc">
              <normaloff>:/icons/icoViewStudy.png</normaloff>:/icons/icoViewStudy.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>266</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lblServerStatusConst">
            <property name="text">
             <string>App Status: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblServerSetup">
            <property name="text">
             <string>Connected to the Server!</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>265</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
